src/ploomber/tasks/notebook.py

```diff
@@ -910,17 +948,31 @@ def __init__(self,
                  params=None,
                  ext_in=None,
                  static_analysis='regular',
-                 local_execution=False):
+                 local_execution=False,
+                 extract_up=False,
+                 extract_prod=False
```

Comment:
same as NotebookRunner, extract_up and extract_prod will always be False so no need to have them here